### ISSUE TITLE
Comments in stored procedure declaration are placed in the arguments

### ISSUE THREAD

**Issue by boubou191911** [darold/ora2pg#1309]
Hello,

I have a procedure where some comments were placed between the CREATE and the AS.
```
CREATE OR REPLACE PROCEDURE EBO_TEST_4(
       ityperun     IN       INTEGER,
       sstartdate   IN       VARCHAR2,
       ireturn      OUT      INTEGER,
       serrormsg    OUT      VARCHAR2
    )
    /* dDAY Format YYYY/MM/DD
    */
    AS 
BEGIN
  NULL;
END EBO_TEST_4;
```
Oracle is ok with this.

When transalated by ora2pg, the result is:
```
-- Generated by Ora2Pg, the Oracle database Schema converter, version 23.0
-- Copyright 2000-2021 Gilles DAROLD. All rights reserved.
-- DATASOURCE: dbi:Oracle:host=

SET client_encoding TO 'UTF8';

\set ON_ERROR_STOP ON



CREATE OR REPLACE PROCEDURE eaidb.ebo_test_4 ( ityperun integer, sstartdate text, ireturn INOUT integer, serrormsg INOUT text  /* dDAY Format YYYY/MM/DD
    */
) AS $body$
BEGIN
  NULL;
END;
$body$
LANGUAGE PLPGSQL
;
-- REVOKE ALL ON PROCEDURE eaidb.ebo_test_4 ( ityperun integer, sstartdate text, ireturn INOUT integer, serrormsg INOUT text   ) FROM PUBLIC;
```
The comment is included in the procedure arguments. And this is not accepted by postgres.

Maybe ora2pg can handle this case.

Regards

**Comment by darold** [comment]
This should be solved by commit 7746541

**Comment by boubou191911** [comment]
Indeed, solved. Thanks!
